.github/workflows/reviewdog.yml

```
@@ -0,0 +1,29 @@
+name: reviewdog
+on:
+ pull_request:
+     branches:
+      - master
+      - adding-review-dog
+
+jobs:
+  reviewdog:
+    name: reviewdog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+        id: go
+      - name: Check out code.
+        uses: actions/checkout@v1
```

```suggestion
        uses: actions/checkout@v3
```
https://github.com/marketplace/actions/checkout#checkout-v3